src/code/z_map_data.c

```diff
@@ -1,79 +1,577 @@
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801094C8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801094F8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109528.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010954C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010956C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010958C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801095AC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801095DC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010960C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109630.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109650.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109670.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801096D4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109714.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109754.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801097C8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010983C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801098A0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109908.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109964.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_8010997C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801099AC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_801099DC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109A00.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109A20.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109A40.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109A98.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109AD8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109B38.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109BA0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109BF4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109C38.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109CBC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109D40.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109DD8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109E70.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109EF8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_data/func_80109F78.s")
+#include "z64map.h"
+#include "assets/objects/gameplay_dangeon_keep/gameplay_dangeon_keep.h"
+
+static Color_RGBA8 mapColorTable[3] = {
```

Comment:
```suggestion
static Color_RGBA8 sMapColorTable[] = {
```
They should also have the `s` prefix